pkg/metrics/metrics.go

```diff
@@ -73,6 +73,11 @@ const (
 	// ClusterStatusCapacityAvailable - metric name for the number of available instances
 	ClusterStatusCapacityAvailable = "cluster_status_capacity_available"
 
+	// KasFleetManagerClusterProviderResourceQuotaConsumed - metric name for how much quota an organisation is currently using
+	KasFleetManagerClusterProviderResourceQuotaConsumed = "kas_fleet_manager_cluster_provider_resource_quota_consumed"
+
+	LabelQuotaId    = "quota_id"
+	LabelProvider   = "provider"
```

Comment:
Let's make this more specific to cluster_provider to differentiate it with other provider types like cloud provider.